book/src/standard-types-and-functions.md

```diff
@@ -17,6 +17,53 @@ When compiling an expression, the compiler automatically inserts a small prelude
 
 Gluon has support for cooperative threading and communication between them through the `Thread` and `Sender`/`Receiver` types.
 
+### Strings
+
+`String` is a built-in data type. The module `std.string` provides the infix operatoin `++`, concatenating two strings. The operation `show` converts the Int to a printable String.
+```
+let io @ { ? } = import! std.io
+let string @ { (++), len } = import! std.string
+let { show }  = import! std.show
+let message : String = "Hello" ++ " " ++ "World" ++ "!"
+let prompt s = "<" ++   show (len s) ++ "> "
+io.println ( (prompt message) ++ message )
+```
+The output will be
+```
+<12> Hello World!
+```
+
+The module `std.string` is providing the following operations
```

Comment:
Could link to the API reference directly instead https://gluon-lang.org/doc/crates_io/std/std/string.html

Reply:
done